### PR TITLE
EXT-X-START tag support

### PR DESCRIPTION
**Pull request by masterful**
This should add support for the [EXT-X-START](https://tools.ietf.org/html/draft-pantos-http-live-streaming-18#section-4.3.5.2) tag and its two attributes.

### CODE REVIEW

File: src/parse-stream.js [hunk 1]
```diff
@@ -363,6 +363,21 @@ export default class ParseStream extends Stream {
...
+    match = (/^#EXT-X-START:?(.*)$/).exec(line);
+    if (match) {
+      event = {
+        type: 'tag',
+        tagType: 'start'
+      };
+      if (match[1]) {
+        event.attributes = parseAttributes(match[1]);
+
+        event.attributes['TIME-OFFSET'] = parseFloat(event.attributes['TIME-OFFSET'], 10);
```
Comment (gesinger) [inline src/parse-stream.js]: Does parseFloat take a second parameter?

File: test/m3u8.test.js [hunk 2]
```diff
@@ -731,6 +731,58 @@ QUnit.test('parses prefixed with 0x or 0X #EXT-X-KEY:IV tags', function(assert)
...
+    '#EXT-X-ENDLIST'
+  ].join('\n');
+  let warning;
+
+  parser.on('warn', function(warn) {
+    warning = warn;
+  });
+  parser.push(manifest);
+
+  assert.ok(warning, 'a warning was triggered');
```
Comment (gesinger) [inline test/m3u8.test.js]: Minor, but might be worth testing the warning message.

File: src/parse-stream.js (hunk 1 above)
Comment (masterful) [reply src/parse-stream.js]: I don't believe it does - sorry about that, I'm used to `parseInt` taking a base parameter and just assumed `parseFloat` behaved similarly...

**Comment by gesinger** [comment]
Tested locally and tests pass after working around rollup issue.

**Pull request merged**